api/Apis/StagingTablesApi.md

```diff
@@ -0,0 +1,36 @@
+# StagingTablesApi
+
+All URIs are relative to *http://localhost:8080/api/2.1/unity-catalog*
+
+| Method | HTTP request | Description |
+|------------- | ------------- | -------------|
+| [**createStagingTable**](StagingTablesApi.md#createStagingTable) | **POST** /staging-tables | Create a staging table |
+
+
+<a name="createStagingTable"></a>
+# **createStagingTable**
+> StagingTableInfo createStagingTable(CreateStagingTable)
+
+Create a staging table
+
+    Creates a new staging table instance. This table must be committed to a regular table before it can be used. 
```

Comment:
Creates a new staging table instance. The `catalog_name.schema_name.name` parameters provided in this request are used to initialize any required storage properties and determine the storage URL that should be used for the data contained by this table. Temporary credentials can be obtained as though the staging table were a regular table.

        When a managed table is created with a URL owned by a staging table, that staging table is "promoted" into the newly created table. This allows for the atomic creation of a table that already has full data written to its storage location. Note: the name provided must match the name used to initialize the staging table originally.